saas/api/metrics.py

```diff
@@ -60,6 +63,40 @@ def get(self, request, *args, **kwargs): #pylint: disable=unused-argument
         return Response(result)
 
 
+class RevenueAPIView(ProviderMixin, APIView):
+    """
+    Produce revenue stats
+    """
+    def get(self, request, *args, **kwargs):
+        ends_at = request.GET.get('ends_at', None)
+        if ends_at:
+            ends_at = parse_datetime(ends_at)
+        ends_at = datetime_or_now(ends_at)
+
+        table_key = request.GET['table_key']
```

Comment:
Following on comments in [previous pull request](https://github.com/djaodjin/djaodjin-saas/pull/56):
For the ``ends_at`` parameter, we implement the convention: _if the query parameter does not exist, use a default_. Unless there is a technical reason, we should implement the same convention for ``table_key``.